vendor/github.com/openshift/library-go/pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -83,18 +140,51 @@ func NewInstallerController(
 func (c *InstallerController) createInstallerController(operatorSpec *operatorv1alpha1.OperatorSpec, originalOperatorStatus *operatorv1alpha1.StaticPodOperatorStatus, resourceVersion string) (bool, error) {
 	operatorStatus := originalOperatorStatus.DeepCopy()
 
+	if len(operatorStatus.NodeStatuses) == 0 {
+		return false, nil
+	}
+
+	// find upgrading node as this will be the first to start new deployment (to minimize number of down nodes)
```

Comment:
@sttts i would probably put this logic into separate function? like `determineNodeToStart()` or something... I think this logic is very important for the operator?